deploy/helm/sumologic/README.md

```diff
@@ -502,6 +502,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `debug.sumologicMock.deployment.tolerations`                                          | Tolerations for the Sumo Logic Mock deployment.                                                                                                                                                                                                                                                            | `{}`                                                                                                                                                                                 |
 | `debug.sumologicMock.enabled`                                                         | Set to `true` to enable Sumo Logic Mock pod.                                                                                                                                                                                                                                                               | `false`                                                                                                                                                                              |
 | `debug.sumologicMock.image`                                                           | Docker image for the Sumo Logic Mock.                                                                                                                                                                                                                                                                      | `{"repository": "public.ecr.aws/sumologic/sumologic-mock", "tag": "2.22.0-59-g245ae92", "pullPolicy": "IfNotPresent"}`                                                               |
+| `debug.enableLocalMode`                                                               | Use it along with `debug.sumologicMock.enabled` in order to install collection locally without accessId, accessKey                                                                                                                                                                                         | `false`                                                                                                                                                                              |
```

Comment:
Will we want to add some options to configure the local mode? If so, then I'd make a key `debug.localMode.enable` instead.

Reply:
I do not plan any for now. For me it is just using mock for registration and collector creation process. Do you have any idea what could be change? Also as experimental and debugging feature we can change it any time :sweat:

Reply:
Ideas to customize in the future:
- only send specific signal to mock (logs/metrics/traces)
- only send to mock vs. send to both mock and Sumo

Reply:
> only send to mock vs. send to both mock and Sumo

This is supported right now

> only send specific signal to mock (logs/metrics/traces)

This also (in scenario when both Sumo Logic and Mock are being used)